test/demo/basic.html

```diff
@@ -21,13 +21,12 @@
     <script type="text/javascript">
 
         var viewer = OpenSeadragon({
-            // debugMode: true,
+            debugMode: true,
```

Comment:
The basic.html file is definitely good for testing, but please return it to its original state before checking in. In this case, please comment this line back out again.